javaslang/src/test/java/javaslang/collection/TreeMapTest.java

```diff
@@ -8,20 +8,26 @@
 
 import javaslang.Tuple;
 import javaslang.Tuple2;
+import javaslang.control.Option;
+import org.assertj.core.api.Assertions;
 import org.junit.Test;
 
 import java.io.Serializable;
 import java.math.BigDecimal;
 import java.util.ArrayList;
 import java.util.Comparator;
 import java.util.HashMap;
+import java.util.NoSuchElementException;
 import java.util.function.Function;
 import java.util.function.Supplier;
 import java.util.stream.Collector;
 
-import static javaslang.collection.Comparators.naturalComparator;
-
 public class TreeMapTest extends AbstractSortedMapTest {
+    private final Tuple2<String, Integer> aaron = Tuple.of("aaron", 5);
```

Comment:
whenever the value of a variable is the same as it's name, I find it a code smell.
It's extracted to a variable, because I should be able to change it (without renaming it also).
Please rename to reflect the intention, not the value :)